components/ILIAS/UI/src/Implementation/Component/Legacy/Renderer.php

```diff
@@ -41,9 +47,25 @@ public function render(Component\Component $component, RendererInterface $defaul
 
         $component = $this->registerSignals($component);
         $this->bindJavaScript($component);
+
+        if ($component->isLatexEnabled()) {
+            return $this->addLatexEnabling($component->getContent());
+        } elseif ($component->isLatexDisabled()) {
+            return $this->addLatexDisabling($component->getContent());
+        }
+
         return $component->getContent();
     }
 
+    /**
+     * @inheritdocs
+     */
```

Comment:
I suggest removing the `inheritdoc` (everywhere), see: https://docs.phpdoc.org/guide/guides/inheritance.html